solutions/python-igraph/visualize_cliques/visualize_cliques_with_edges.py

```diff
@@ -0,0 +1,27 @@
+import igraph as ig
+import matplotlib.pyplot as plt
+
+g = ig.Graph.Famous('Zachary')
+g.vs['label'] = [i + 1 for i in range(g.vcount())]
+g.vs['size'] = 0
+g.es['width'] = 0.3
+g.vs['color'] = 'yellow'
+
+fig, ax = plt.subplots(3, 4)
+cliques = g.cliques(4, 4)
+for i in range(len(cliques)):
```

Comment:
Same here, you can use
```suggestion
for i, clique in enumerate(cliques):
```